drivers/applications/driver/driver.go

```diff
@@ -0,0 +1,96 @@
+package driver
+
+import (
+	"context"
+
+	. "github.com/portworx/torpedo/drivers/applications/apptypes"
+	. "github.com/portworx/torpedo/drivers/applications/mysql"
+	. "github.com/portworx/torpedo/drivers/applications/postgres"
+	. "github.com/portworx/torpedo/drivers/utilities"
+)
+
+type ApplicationDriver interface {
```

Comment:
We need to create a default application in case we are going to run with an application which does not have the Application Driver implementation yet.
We need to make sure, if I run with any other app like `fio`, it is not going to break anything

Reply:
We are only creating handlers for the apps which have data support set to true, so if an app do not explicity have data injection support app handler will not be created for the same if appInfo.StartDataSupport is the condition

Reply:
We rely on below service annotation to check if the data injection support is present or not

  annotations:
    startDataSupported: "true"
    username: "root"
    password: "password"
    databaseName: "mysql"
    port: "3306"
    appType: "mysql"